### ISSUE TITLE
goShimmer node does not sync after complete reinstallation

### ISSUE THREAD

**Issue by stricki** [iotaledger/goshimmer#689]
## Bug description

After reinstalling my goshimmer nodes (v0.2.2.) I notice that both of my nodes do not fully sync anymore. I have restartet the installation several times and even with a reinstallation of ubuntu 20.04 LTS. But interestingly they get the beacons only until about the last time they were synced before... I wonder if this problem is known, because I have also installed goshimmer nodes on other VPS and they sync quite fast (within 10 - 15 min)

## GoShimmer version

- GoShimmer version: 0.2.2

## Hardware specification

- Operating system: Ubuntu 20.04 LTS
- RAM: 8 GB
- Number of cores: 4
- Device type: VPS at Contabo

## Steps To reproduce the bug

Explain how the maintainer can reproduce the bug.

1. Reinstall the VPS (keep the same IP address)
2. Follow the official installation tutorial here: https://github.com/iotaledger/goshimmer/wiki/Setup-up-a-GoShimmer-node-(Joining-the-pollen-testnet)

## Expected behaviour
I would expect that the node will sync in a couple of minutes (15 - 20 minutes), once it is up and running

## Actual behaviour

The node does not sync anymore. It gets the sync beacons from about when it was synced for the last time but no progress from there. The nodes were not synced even after giving them several hours to do so...

## Errors

Paste any errors that you see, including logs, errors, or screenshots.
8080/info of the first node:
```
{
  "version": "v0.2.2",
  "synced": false,
  "beacons": [
    {
      "public_key": "Gm7W191NDnqyF7KJycZqK7V6ENLwqxTwoKQN4SmpkB24",
      "msg_id": "3ya8cMKgcYjXEdZMy8cnCfxsHfUKMEGey62ZYEaqJQsapScgDsKz4ZpHgD7A9jY55JQADyiXT6xuPAMPNBfSUvrq",
      "sent_time": 1595860060364146000,
      "synced": false
    },
    {
      "public_key": "9DB3j9cWYSuEEtkvanrzqkzCQMdH1FGv3TawJdVbDxkd",
      "msg_id": "35tVZDrTNzggjofetYSk4tHk4YfCEeVmTxeJaQpz8xUb25AYxNxKx4XkRM6oDqvcHTHqFRsx3qMKEG7J3YB6HgyD",
      "sent_time": 1595860059080045800,
      "synced": false
    }
  ],
  "identityID": "RGCewNmbtuc",
  "publicKey": "JCU1EXzR7AJXXwbYg3bHBjgTTyUbcZAB7eTViZVKEJtR",
  "messageRequestQueueSize": 859,
  "solidMessageCount": 996,
  "totalMessageCount": 422925,
  "enabledPlugins": [
    "Analysis-Client",
    "Autopeering",
    "CLI",
    "Config",
    "DRNG",
    "Dashboard",
    "Database",
    "Gossip",
    "Graceful Shutdown",
    "Issuer",
    "Logger",
    "MessageLayer",
    "Metrics",
    "NetworkDelay",
    "PoW",
    "PortCheck",
    "Profiling",
    "Prometheus",
    "RemoteLog",
    "Spammer",
    "SyncBeaconFollower",
    "ValueTransfers",
    "WebAPI",
    "WebAPI DRNG Endpoint",
    "WebAPI Value Endpoint",
    "WebAPI autopeering Endpoint",
    "WebAPI data Endpoint",
    "WebAPI faucet Endpoint",
    "WebAPI healthz Endpoint",
    "WebAPI info Endpoint",
    "WebAPI message Endpoint"
  ],
  "disabledPlugins": [
    "Analysis-Dashboard",
    "Analysis-Server",
    "Banner",
    "Faucet",
    "SyncBeacon",
    "WebAPI Auth",
    "WebAPI tools Endpoint"
  ]
}
```
![Screenshot from 2020-08-01 13-34-11](https://user-images.githubusercontent.com/17906700/89100959-12de4f80-d3fc-11ea-8ade-d71b2321ca5f.png)

8080/info of the second node:
```
{
  "version": "v0.2.2",
  "synced": false,
  "beacons": [
    {
      "public_key": "9DB3j9cWYSuEEtkvanrzqkzCQMdH1FGv3TawJdVbDxkd",
      "msg_id": "2twJ6C9m98QkP1TL8NZ6xoCmoVYkAvxB4FNza2SceR8vAdUVT3CNY7baptd3mdt6t7J5fC9aBeTRebY8mEZBtS5G",
      "sent_time": 1596176859080023800,
      "synced": false
    },
    {
      "public_key": "Gm7W191NDnqyF7KJycZqK7V6ENLwqxTwoKQN4SmpkB24",
      "msg_id": "5GgPr6MWKV35wNRQ34TqZnRDnMTqNjnvyVXUn64nA8QPCqWMhkxPAaEyv9FsNXVYxkvDvxBo7Ahu8LqFQFhowCcd",
      "sent_time": 1596176860364122400,
      "synced": false
    }
  ],
  "identityID": "RJFHGWi5fo1",
  "publicKey": "3eddreAaYeTJnCpyBwcWU9EemDDfw7monH1tsjtSrLcV",
  "messageRequestQueueSize": 189,
  "solidMessageCount": 313637,
  "totalMessageCount": 435501,
  "enabledPlugins": [
    "Analysis-Client",
    "Autopeering",
    "CLI",
    "Config",
    "DRNG",
    "Dashboard",
    "Database",
    "Gossip",
    "Graceful Shutdown",
    "Issuer",
    "Logger",
    "MessageLayer",
    "Metrics",
    "NetworkDelay",
    "PoW",
    "PortCheck",
    "Profiling",
    "Prometheus",
    "RemoteLog",
    "Spammer",
    "SyncBeaconFollower",
    "ValueTransfers",
    "WebAPI",
    "WebAPI DRNG Endpoint",
    "WebAPI Value Endpoint",
    "WebAPI autopeering Endpoint",
    "WebAPI data Endpoint",
    "WebAPI faucet Endpoint",
    "WebAPI healthz Endpoint",
    "WebAPI info Endpoint",
    "WebAPI message Endpoint"
  ],
  "disabledPlugins": [
    "Analysis-Dashboard",
    "Analysis-Server",
    "Banner",
    "Faucet",
    "SyncBeacon",
    "WebAPI Auth",
    "WebAPI tools Endpoint"
  ]
}
```
![Screenshot from 2020-08-01 13-34-45](https://user-images.githubusercontent.com/17906700/89100964-1b368a80-d3fc-11ea-9f4b-c27dd096af02.png)

**Comment by wonkytonky2** [comment]
same issue

**Comment by svenger87** [comment]
Yup. Same here. Fresh node is not able to sync.

**Comment by alorraine** [comment]
I've found the same issue today, after a fresh install on Hetzner with the same IP address.
[log.txt](https://github.com/iotaledger/goshimmer/files/5021146/log.txt)

**Comment by TeeVeeEss** [comment]
@alorraine your problem is that udp port 14626 is not open, at least that can be seen in your log file:
```2020-08-04T10:40:03+02:00	WARN	PortCheck	portcheck/plugin.go:75	Error pinging entry node 116.202.49.178: response timeout```
```2020-08-04T10:40:03+02:00	FATAL	PortCheck	portcheck/plugin.go:79	Please check that GoShimmer is publicly reachable at port 14626/udp```

**Comment by stricki** [comment]
Today I have set up a new goshimmer node on a VPS with an IP that at least I have never used for goshimmer before. I received the IP yesterday together with my new VPS. So maybe the sync problem is a more general problem ant not tied to the reuse of IP after a complete new installation of the VPS. After it didn't sync for hours I did a reset:

$ docker-compose down
$ rm db/*
$ docker-compose pull
$ docker-compose up -d

and the log.txt from that point is attached

[log.txt](https://github.com/iotaledger/goshimmer/files/5029077/log.txt)
